docs/user_guide/authenticate.md

```diff
@@ -1,11 +1,139 @@
 # Authentication
 
-Introduces the `earthaccess.login` method for managing Earthdata Login and cloud credentials.
+You can use `earthaccess` to search for datasets and data without needing to login.  However, to access (download or stream) NASA Earth science data, whether from one of the NASA
+Distributed Active Archive Centers (DAACs) or from NASA Earthdata Cloud, you need
+an Earthdata Login.  You can register for a a free Earthdata Login (EDL) account [here](https://urs.earthdata.nasa.gov/).  
 
-!!! note "This Page is a Work in Progress"
+Once you have an Earthdata Login, the `earthaccess.login` method manages Earthdata Login, and cloud credentials when you are working with cloud-hosted data.
 
-    We are reorganizing and updating the documentation, so not all pages are complete.  If you are looking for information about authenticating using earthaccess see the
-    How-Tos and Tutorials in links below.
+`earthaccess.login` offers three methods of logging in (or authenticating) using EDL: [a manual login method](#login-manually), where you enter EDL username and password manually; and two automatic login methods using EDL credentials stored in a [`.netrc`](#login-using-a-netrc) file or in [environment variables](#login-using-environment-variables).  By default, `earthaccess.login()` will look for a `.netrc` or environment variables first.  If neither of these are found, it will prompt you to enter your username and password.  The three methods are described in detail below.  
 
-    * [Quick start](../quick-start.md)
-    * [How-To Authenticate with earthaccess](../howto/authenticate.md)
+`earthaccess.login` can also be used to login to [different endpoints](#accessing-different-endpoints) and [get S3 credentials](#using-earthaccess-to-get-credentials).
+
+## Login Manually
+
+If you have not created a `.netrc` file or `EARTHDATA_USERNAME` and `EARTHDATA_PASSWORD`, you can use the following approach to login.
+
+```
+>>> import earthaccess
+
+>>> auth = earthaccess.login()
+Enter your Earthdata Login username: your_username
+Enter your Earthdata password: 
+```
+
+You don't need to assign the result of `earthaccess.login()` to a variable but doing so enables access session information.  These are discussed in [Accessing Session Information]().
+
+Setting `strategy=interactive` will force a manual login.
+
+## Login using a `.netrc`
```

Comment:
I think a note here might be really useful -- we don't (yet) support encrypted netrc files, and this strategy of writing credentials in plain text to disk **should not** be used on untrusted machines or shared user accounts.